docs/core/linux-prerequisites.md

```diff
@@ -194,59 +157,13 @@ Enable .NET Core 1.0 SDK and Runtime for your environment:
 
 For Red Hat .NET channel access registration help, see [Chapter 1 of the .NET Core 1.1 Getting Started Guide](https://access.redhat.com/documentation/en/net-core/1.1/paged/getting-started-guide/) at Red Hat.
 
-## Install .NET Core for Ubuntu 14.04, Ubuntu 16.04, Ubuntu 16.10 & Linux Mint 17, Linux Mint 18 (64 bit)
+## Install .NET Core for supported Ubuntu and Linux Mint distributions\versions (64 bit)
 
 1. Remove any **previous preview** versions of .NET Core from your system.
 
 # [.NET Core 2.x](#tab/netcore2x)
 
-2. Register the Microsoft Product key as trusted.
-
-   ```bash
-   curl https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > microsoft.gpg
-   sudo mv microsoft.gpg /etc/apt/trusted.gpg.d/microsoft.gpg
-   ```
-
-3. Set up the desired version host package feed.
-
-   **Ubuntu 17.10**
-
-   ```bash
-   sudo sh -c 'echo "deb [arch=amd64] https://packages.microsoft.com/repos/microsoft-ubuntu-artful-prod artful main" > /etc/apt/sources.list.d/dotnetdev.list'
-   sudo apt-get update
-   ```
-   **Ubuntu 17.04**
-
-   ```bash
-   sudo sh -c 'echo "deb [arch=amd64] https://packages.microsoft.com/repos/microsoft-ubuntu-zesty-prod zesty main" > /etc/apt/sources.list.d/dotnetdev.list'
-   sudo apt-get update
-   ```
-
-   **Ubuntu 16.04 / Linux Mint 18**
-
-   ```bash
-   sudo sh -c 'echo "deb [arch=amd64] https://packages.microsoft.com/repos/microsoft-ubuntu-xenial-prod xenial main" > /etc/apt/sources.list.d/dotnetdev.list'
-   sudo apt-get update
-   ```
-
-   **Ubuntu 14.04 / Linux Mint 17**
-
-   ```bash
-   sudo sh -c 'echo "deb [arch=amd64] https://packages.microsoft.com/repos/microsoft-ubuntu-trusty-prod trusty main" > /etc/apt/sources.list.d/dotnetdev.list'
-   sudo apt-get update
-   ```
-
-4. Install .NET Core.
-
-   ```bash
-   sudo apt-get install dotnet-sdk-2.1.3
-   ```
-
-4. Run the `dotnet --version` command to prove the installation succeeded.
-
-   ```bash
-   dotnet --version
-   ```
+2. To ensure you have the latest installation information, use the [.NET Core 2.x SDK and Runtime Installer instructions](https://www.microsoft.com/net/learn/get-started/linuxubuntu) for supported Ubuntu and Linux Mint distributions\versions (64 bit).
```

Comment:
mixing steps in and out of the tab makes it hard to follow. I almost thought it was a mistake to start at step 2 when I noticed you had step 1 before the tab.